--- kaffeine-0.8.1/kaffeine/src/kaffeine.desktop~	2006-04-04 14:13:46.488028381 +0300
+++ kaffeine-0.8.1/kaffeine/src/kaffeine.desktop	2006-04-04 14:15:39.320534900 +0300
@@ -34,7 +34,7 @@
 GenericName[nl]=Mediaspeler
 GenericName[nn]=Mediespelar
 GenericName[pa]=ਵੀਡਿਓ ਪਲੇਅਰ
-GenericName[pl]=Odtwarzacz multimedialny
+GenericName[pl]=Odtwarzacz multimediów
 GenericName[pt]=Leitor Multimédia
 GenericName[pt_BR]=Reprodutor Multimídia
 GenericName[sr]=Медија плејер
--- kaffeine-0.8.1/kaffeine/src/kaffeine.desktop~	2006-04-04 14:15:39.320534900 +0300
+++ kaffeine-0.8.1/kaffeine/src/kaffeine.desktop	2006-04-04 14:16:16.621363587 +0300
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Categories=Qt;KDE;Application;AudioVideo;Player;
+Categories=Qt;KDE;AudioVideo;Player;
 Comment=Version 0.8
 Comment[bg]=Версия 0.8
 Comment[br]=Stumm 0.8
--- kaffeine-0.8.1/kaffeine/src/kaffeine.desktop~	2006-04-04 14:16:16.621363587 +0300
+++ kaffeine-0.8.1/kaffeine/src/kaffeine.desktop	2006-04-04 14:18:27.014260694 +0300
@@ -1,15 +1,15 @@
 [Desktop Entry]
 Categories=Qt;KDE;AudioVideo;Player;
-Comment=Version 0.8
-Comment[bg]=Версия 0.8
-Comment[br]=Stumm 0.8
-Comment[el]=Έκδοση 0.8
-Comment[ga]=Leagan 0.8
-Comment[it]=Versione 0.8
-Comment[ja]=バージョン 0.8
-Comment[nl]=Versie 0.8
-Comment[pt]=Versão 0.8
-Comment[pt_BR]=Versão 0.8
+Comment=Media Player
+Comment[bg]=Медия плеър
+Comment[br]=Soner liesvedia
+Comment[el]=Αναπαραγωγέας πολυμέσων
+Comment[ga]=Seinnteoir Meán
+Comment[it]=Lettore multimediale
+Comment[ja]=メディアプレーヤ
+Comment[nl]=Mediaspeler
+Comment[pt]=Leitor Multimédia
+Comment[pt_BR]=Reprodutor Multimídia
 DocPath=kaffeine/index.html
 Encoding=UTF-8
 Exec=kaffeine %U
