--- kaffeine-0.4beta1/Makefile.am.orig	2003-09-12 12:09:16.000000000 +0200
+++ kaffeine-0.4beta1/Makefile.am	2003-10-20 15:42:31.000000000 +0200
@@ -1,6 +1,6 @@
 ####### kdevelop will overwrite this part!!! (begin)##########
 
-SUBDIRS = kaffeine po doc misc addons 
+SUBDIRS = kaffeine po misc addons 
 
 EXTRA_DIST = kaffeine.kdevprj admin AUTHORS COPYING ChangeLog INSTALL README TODO kaffeine.lsm LIESMICH 
 
--- kaffeine-0.4beta1/Makefile.in.orig	2003-09-24 19:06:31.000000000 +0200
+++ kaffeine-0.4beta1/Makefile.in	2003-10-20 15:42:49.000000000 +0200
@@ -250,7 +250,7 @@
 x_includes = @x_includes@
 x_libraries = @x_libraries@
 
-SUBDIRS = kaffeine po doc misc addons 
+SUBDIRS = kaffeine po misc addons 
 
 EXTRA_DIST = kaffeine.kdevprj admin AUTHORS COPYING ChangeLog INSTALL README TODO kaffeine.lsm LIESMICH 
 
