diff -urN kaffeine-0.4.1.orig/Makefile.am kaffeine-0.4.1/Makefile.am
--- kaffeine-0.4.1.orig/Makefile.am	2004-01-12 21:57:02.000000000 +0100
+++ kaffeine-0.4.1/Makefile.am	2004-02-05 23:01:48.452527317 +0100
@@ -1,6 +1,6 @@
 
 
-SUBDIRS = kaffeine po doc misc addons mimetypes protocols
+SUBDIRS = kaffeine po misc
 
 EXTRA_DIST = kaffeine.kdevprj admin AUTHORS COPYING ChangeLog INSTALL README TODO kaffeine.lsm BUGS faq.html
 
