--- ./po/Makefile.in~	2005-09-05 01:46:48.000000000 +0300
+++ ./po/Makefile.in	2005-09-05 01:48:20.000000000 +0300
@@ -315,8 +315,8 @@
 ####### kdevelop will overwrite this part!!! (end)############
 #>- POFILES = AUTO
 #>+ 2
-POFILES = zh_CN.po ca.po da.po de.po cs.po es.po fi.po fr.po gl.po he.po hu.po ja.po it.po mk.po nb.po nl.po nn.po pl.po ru.po sv.po tr.po pt_BR.po pt_PT.po
-GMOFILES = zh_CN.gmo ca.gmo da.gmo de.gmo cs.gmo es.gmo fi.gmo fr.gmo gl.gmo he.gmo hu.gmo ja.gmo it.gmo mk.gmo nb.gmo nl.gmo nn.gmo pl.gmo ru.gmo sv.gmo tr.gmo pt_BR.gmo pt_PT.gmo
+POFILES = zh_CN.po ca.po da.po de.po cs.po es.po fi.po fr.po gl.po he.po hu.po ja.po it.po mk.po nb.po nl.po nn.po pl.po ru.po sv.po tr.po pt_BR.po pt.po
+GMOFILES = zh_CN.gmo ca.gmo da.gmo de.gmo cs.gmo es.gmo fi.gmo fr.gmo gl.gmo he.gmo hu.gmo ja.gmo it.gmo mk.gmo nb.gmo nl.gmo nn.gmo pl.gmo ru.gmo sv.gmo tr.gmo pt_BR.gmo pt.gmo
 EXTRA_DIST = kaffeine.pot
 #>- all: all-am
 #>+ 1
@@ -511,7 +511,7 @@
 
 
 #>+ 2
-KDE_DIST=tr.gmo mk.po zh_CN.po nl.po sv.gmo pl.gmo nn.po ru.po da.po ru.gmo he.gmo nb.gmo fi.po gl.po de.po fr.gmo ja.gmo nb.po fr.po ca.po cs.po mk.gmo zh_CN.gmo tr.po he.po cs.gmo pt_BR.gmo nn.gmo pt_PT.gmo Makefile.in pt_PT.po pt_BR.po ja.po it.gmo sv.po fi.gmo hu.po de.gmo pl.po es.po es.gmo gl.gmo it.po nl.gmo hu.gmo Makefile.am da.gmo ca.gmo 
+KDE_DIST=tr.gmo mk.po zh_CN.po nl.po sv.gmo pl.gmo nn.po ru.po da.po ru.gmo he.gmo nb.gmo fi.po gl.po de.po fr.gmo ja.gmo nb.po fr.po ca.po cs.po mk.gmo zh_CN.gmo tr.po he.po cs.gmo pt_BR.gmo nn.gmo pt.gmo Makefile.in pt.po pt_BR.po ja.po it.gmo sv.po fi.gmo hu.po de.gmo pl.po es.po es.gmo gl.gmo it.po nl.gmo hu.gmo Makefile.am da.gmo ca.gmo 
 
 #>+ 70
 zh_CN.gmo: zh_CN.po
@@ -580,17 +580,17 @@
 pt_BR.gmo: pt_BR.po
 	rm -f pt_BR.gmo; $(GMSGFMT) -o pt_BR.gmo $(srcdir)/pt_BR.po
 	test ! -f pt_BR.gmo || touch pt_BR.gmo
-pt_PT.gmo: pt_PT.po
-	rm -f pt_PT.gmo; $(GMSGFMT) -o pt_PT.gmo $(srcdir)/pt_PT.po
-	test ! -f pt_PT.gmo || touch pt_PT.gmo
+pt.gmo: pt.po
+	rm -f pt.gmo; $(GMSGFMT) -o pt.gmo $(srcdir)/pt.po
+	test ! -f pt.gmo || touch pt.gmo
 
 #>+ 3
 clean-nls:
-	-rm -f  zh_CN.gmo ca.gmo da.gmo de.gmo cs.gmo es.gmo fi.gmo fr.gmo gl.gmo he.gmo hu.gmo ja.gmo it.gmo mk.gmo nb.gmo nl.gmo nn.gmo pl.gmo ru.gmo sv.gmo tr.gmo pt_BR.gmo pt_PT.gmo
+	-rm -f  zh_CN.gmo ca.gmo da.gmo de.gmo cs.gmo es.gmo fi.gmo fr.gmo gl.gmo he.gmo hu.gmo ja.gmo it.gmo mk.gmo nb.gmo nl.gmo nn.gmo pl.gmo ru.gmo sv.gmo tr.gmo pt_BR.gmo pt.gmo
 
 #>+ 10
 install-nls:
-	@for base in zh_CN ca da de cs es fi fr gl he hu ja it mk nb nl nn pl ru sv tr pt_BR pt_PT ; do \
+	@for base in zh_CN ca da de cs es fi fr gl he hu ja it mk nb nl nn pl ru sv tr pt_BR pt ; do \
 	  echo $(INSTALL_DATA) $$base.gmo $(DESTDIR)$(kde_locale)/$$base/LC_MESSAGES/$(PACKAGE).mo ;\
 	  $(mkinstalldirs) $(DESTDIR)$(kde_locale)/$$base/LC_MESSAGES ; \
 	  if test -f $$base.gmo; then $(INSTALL_DATA) $$base.gmo $(DESTDIR)$(kde_locale)/$$base/LC_MESSAGES/$(PACKAGE).mo ;\
@@ -623,7 +623,7 @@
 	rm -f $(DESTDIR)$(kde_locale)/sv/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(kde_locale)/tr/LC_MESSAGES/$(PACKAGE).mo
 	rm -f $(DESTDIR)$(kde_locale)/pt_BR/LC_MESSAGES/$(PACKAGE).mo
-	rm -f $(DESTDIR)$(kde_locale)/pt_PT/LC_MESSAGES/$(PACKAGE).mo
+	rm -f $(DESTDIR)$(kde_locale)/pt/LC_MESSAGES/$(PACKAGE).mo
 
 #>+ 2
 all-nls: $(GMOFILES)
